memberships/static/memberships/css/main.css

```diff
@@ -1,66 +1,193 @@
-:root {
-  /* alternative-yellow: #FFDC00; */
-  --yellow: #fff000;
-  --page-width: 1280px;
-}
+/***** General styles *****/
 
 body {
+  --content-width: 1080px;
+  --text-spacing: .5rem;
+  
+  /* default colors (for light mode) */
+  --selection-bg: #FF0000;
+  --selection-color: #FFF;
+  --a-color: #0033CC;
+  --button-bg: #FFFF00;
+  --button-bg-hover: #FFED00;
+  --button-box-shadow: #555;
+  --button-color: #111;
+  --input-file-border: #111;
+  --helptext-color: #888;
+  --header-bg: #FFFF00;
+  --header-color: #111;
+  --header-a-hover: #FF0000;
+  
   margin: 0;
   font-family: "Helvetica", "Arial", sans-serif;
+  background: #FFF;
   color: #555;
 }
-strong, a {
-  color: var(--yellow);
+body.dark-mode {
+  /* color changes for dark mode */
+  --a-color: #809FFF;
+  --button-box-shadow: #000;
+  --input-file-border: #000;
+  --header-bg: #111;
+  --header-color: #FFFF00;
+
+  background: #333;
+  color: #AAA;
+}
+
+header, main, footer {
+  max-width: var(--content-width);
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+::selection {
+  background: var(--selection-bg);
+  color: var(--selection-color);
+}
+
+a {
+  color: var(--a-color);
+}
+
+.button {
+  margin: 5px 0;
+  padding: .5em 1em;
+  border: none;
+  border-radius: 4px;
+  background: var(--button-bg);
+  box-shadow: 1px 1px 2px var(--button-box-shadow);
+  color: var(--button-color);
+}
+.button:hover {
+  background: var(--button-bg-hover);
+  cursor: pointer;
+}
+.button:active {
+  position: relative;
+  top: 1px;
+  left: 1px;
+  box-shadow: none; 
+}
+
+form table {
+  max-width: 800px;
+}
+
+form th, form td {
+  padding-bottom: var(--text-spacing);
+  vertical-align: top;
+  text-align: left;
+}
+
+form th {
+  padding-right: var(--text-spacing);
+}
+
+form td {
+  display: grid;
+  grid-template-columns: auto auto 1fr;
+}
+form td * {
+  align-self: start;
+}
+form td input:not(:only-child) {
+  margin-bottom: var(--text-spacing);
+  margin-right: var(--text-spacing);
+}
+/*form td input:required::after {
+  content: "*";
+  color: #FF0000;
+}*/
```

Comment:
We may as well remove this commented out code :) Unless we want to keep it?

```suggestion
```

Reply:
Yeah, that's fine - turns out ::after doesn't work with input tags anyway.